scully/renderPlugins/contentRenderPlugin.ts

```diff
@@ -62,6 +53,8 @@ async function handleFile(extension: string, fileContent: string) {
 }
 
 function getScript() {
-  return `<script>try {window['scullyContent'] = document.body.innerHTML.split('<!--scullyContent-begin-->')[1].split('<!--scullyContent-end-->')[0];} catch(e) {console.error('scully could not pare content',e);}</script>
-`;
+  return `<script>
```

Comment:
This was a single line on purpose.  Its a bit more readable like this, I agree, but its not meant to be read ;)
I wrote it to be as small as possible.